recipes-rubygems/rubygems-aws-sdk-core_3.112.0.bb

```diff
@@ -6,14 +6,19 @@ HOMEPAGE = "https://github.com/aws/aws-sdk-ruby"
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
 
-DEPENDS_class-native += "rubygems-aws-eventstream-native \
-                         rubygems-aws-partitions-native \
-                         rubygems-aws-sigv4-native \
-                         rubygems-jmespath-native"
-RDEPENDS_${PN}_class-target += "rubygems-aws-eventstream \
-                                rubygems-aws-partitions \
-                                rubygems-aws-sigv4 \
-                                rubygems-jmespath"
+DEPENDS_class-native += " \
+	rubygems-aws-eventstream-native \
```

Comment:
I'm not sure if that is a tab (can't really see that in GH) - but I'd like to keep with 4-spaces and no tabs